### ISSUE TITLE
Add the input section of the feed component

### ISSUE THREAD

**Issue by Bungycode** [Bungycode/twitter#8]
### Summary
Create the input section of the feed component to imitate twitter's.

### Tasks
List of things that are involved with accomplishing this 

- [x] Create mini-profile picture.
- [x] Create text box for writing text.
- [x] Create icons to be used for image uploading and emotes.
- [x] Create tweet button.
- [x] Style input section with Tailwind CSS.
- [x] Use third party package @tailwindcss/forms for text box. May need plug-in.

### Acceptance Criteria
Checklist of functionality that must be true for this to be considered done, and things that should be verified when reviewing PRs for this issue.

- [x] Input section resembles twitter's.
- [x] Successful implementation of third party package/plug-in.
- [x] Merge into development branch.

### Blocks
Links to all issues that this issue blocks, and any needed explanation
* ...

### Blocked By
Links to all issues that this issue is dependent on, and any needed explanation
* ... 

> Include Any attachments or links to resources needed to complete this task.
> Link to target repo, assign feature, and labels as necessary.
